src/plugins/tabs/_print.scss

```diff
@@ -18,7 +18,9 @@
 
 	[role="tabpanel"] {
 		display: block !important;
+		max-height: none !important;
```

Comment:
There might be value in doing something like this for ``max-height`` (to avoid needing to resort to ``!important``):

```
.wb-tabs {
	[role="tabpanel"] {
		...

		&.noheight {
			max-height: none;
		}
	}
}
```